tests/particles/particle_handler_07.with_p4est=true.mpirun=1.output

```diff
@@ -31,37 +31,7 @@ DEAL:2d/2d::Particle id 8 is in cell 2.15
 DEAL:2d/2d::     at location 0.850000 0.850000
 DEAL:2d/2d::     at reference location 0.400000 0.400000
 DEAL:2d/2d::OK
-DEAL:2d/3d::Particle number: 10
-DEAL:2d/3d::Particle id 2 is in cell 2.0
-DEAL:2d/3d::     at location 0.250000 0.250000 0.250000
-DEAL:2d/3d::     at reference location 1.00000 1.00000
-DEAL:2d/3d::Particle id 1 is in cell 2.0
-DEAL:2d/3d::     at location 0.150000 0.150000 0.150000
-DEAL:2d/3d::     at reference location 0.600000 0.600000
-DEAL:2d/3d::Particle id 0 is in cell 2.0
-DEAL:2d/3d::     at location 0.0500000 0.0500000 0.0500000
-DEAL:2d/3d::     at reference location 0.200000 0.200000
-DEAL:2d/3d::Particle id 4 is in cell 2.3
-DEAL:2d/3d::     at location 0.450000 0.450000 0.450000
-DEAL:2d/3d::     at reference location 0.800000 0.800000
-DEAL:2d/3d::Particle id 3 is in cell 2.3
-DEAL:2d/3d::     at location 0.350000 0.350000 0.350000
-DEAL:2d/3d::     at reference location 0.400000 0.400000
-DEAL:2d/3d::Particle id 7 is in cell 2.12
-DEAL:2d/3d::     at location 0.750000 0.750000 0.750000
-DEAL:2d/3d::     at reference location 1.00000 1.00000
-DEAL:2d/3d::Particle id 6 is in cell 2.12
-DEAL:2d/3d::     at location 0.650000 0.650000 0.650000
-DEAL:2d/3d::     at reference location 0.600000 0.600000
-DEAL:2d/3d::Particle id 5 is in cell 2.12
-DEAL:2d/3d::     at location 0.550000 0.550000 0.550000
-DEAL:2d/3d::     at reference location 0.200000 0.200000
-DEAL:2d/3d::Particle id 9 is in cell 2.15
-DEAL:2d/3d::     at location 0.950000 0.950000 0.950000
-DEAL:2d/3d::     at reference location 0.800000 0.800000
-DEAL:2d/3d::Particle id 8 is in cell 2.15
-DEAL:2d/3d::     at location 0.850000 0.850000 0.850000
-DEAL:2d/3d::     at reference location 0.400000 0.400000
+DEAL:2d/3d::Particle number: 0
```

Comment:
Do you know what changed here?

Reply:
Yes. The previous algorithm was returning garbage. Particles don't work (yet) properly on manifolds of co-dimension one or two, since in those cases the transformation from real to unit cell is not invertible in general, but just on the manifold, and our algorithms are not robust for this case. The current algorithm won't find particles because it starts off by checking inside a bounding box of the cell first, and all particles here are away from the manifold.